### PR TITLE
Task/bookmark generator

### LINKED ISSUE #68: Add bookmark generator
Something we can use to help with testing sync.

### PR DESCRIPTION
**Pull request by tublitzed**
## Description
Although we're not using it for the quota work, I figured I'd check this in anyways since it might be useful down the road - adding a bookmark generator. While I was in here I also upgraded the one minor sec vulnerability caught by npm audit and added JS linters with a `--fix` flag.

## Testing

Try it out by following the newly added details in the README. Does it work?

## Issue(s)

Closes https://github.com/mozilla-services/services-engineering/issues/68

### CODE REVIEW

File: scripts/bookmarks/README.md [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
 - $ `npm i` - install dependencies.
+
+### Counting
 - [Export](https://support.mozilla.org/en-US/kb/restore-bookmarks-from-backup-or-move-them) your Firefox bookmarks as JSON.
 - Then replace the [bookmarks/samples/default.json](https://github.com/mozilla-services/services-engineering/blob/task/add-bookmark-debugging-script/scripts/bookmarks/samples/default.json) file with your bookmarks.
 - \$ `node scripts/bookmarks/` - this will give you details around dataset size and bookmark + folder counts.
 - \$ `node scripts/bookmarks/ --mock` - this will give dataset size info, bookmark + folder counts AND create an fresh copy for you to use called `mocked_bookmarks.json` which will replace your data with example data. Note: some folders and bookmarks are hidden so while you will note each new mock folder + bookmark is given a unique numeric suffix, you might be missing a few of them if you import into the browser. This is expected.
 
+### Generating
+- \$ `node scripts/bookmarks/ -g` - This will default to generating a JSON file with 10k bookmarks (which is roughly 3MB pre-encryption).
```
Comment (tublitzed) [inline scripts/bookmarks/README.md:17]: Originally I started trying much much higher numbers than this. However, there's a limit to what nightly can import and this is safely below it.

**Review by jrconlin — APPROVED** [review]
I wonder if it might be worth adding an option to generate based on the size of the desired output? This could be either estimated by taking the length of the serialized bookmark field * 1.3 (to approximate base64), or we could just report the size of the generated file.

**Comment by tublitzed** [comment]
@jrconlin I think that's a great suggestion. In the interest of time (or rather, lack thereoff) I'm going to punt that to a future iteration of this, but if it turns out we need this for something again soon I'll hop back in and add it.

**Comment by jrconlin** [comment]
Yeah, totally. I like your approach. Far better than the insanity I came up with: https://github.com/mozilla-services/syncstorage-rs/pull/823

**Pull request merged**